HISTORY.rst

```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+v0.7.2 (TBA)
+------------
+
+* ignore any exception if errors='ignore'
```

Comment:
hey! would you mind making it a bit more detailed, e.g. which function is this argument for?

Reply:
@kmike hope now is more clear.